docs/features/software-catalog/catalog-customization.md

```diff
@@ -178,7 +178,7 @@ const routes = (
     {/* highlight-remove-next-line */}
     <Route path="/catalog" element={<CatalogIndexPage />} />
     {/* highlight-add-start */}
-    <Route path="/catalog" element={<CatalogIndexPage />}>
+    <Route path="/catalog" element={<CustomCatalogPage />}>
```

Comment:
The existing code is actually correct here, we need to use the index page so that routes are bound correctly, the custom page will be provided as an outlet instead and used over here: https://github.com/backstage/backstage/blob/7e8f10e19985454ddcb43c724f33bf7ce80b052c/plugins/catalog/src/components/CatalogPage/CatalogPage.tsx#L25

Reply:
understandable